hasher-matcher-actioner/src/OpenMediaMatch/blueprints/matching.py

```diff
@@ -39,7 +39,7 @@
 class MatchWithDistance(t.TypedDict):
     content_id: int
     distance: str
-
+    banks: list[str]
```

Comment:
blocking: A content_id can only ever be in one bank, so I think this should be similar.